tasks.py

```diff
@@ -32,6 +33,11 @@ def flake(ctx):
     ctx.run('flake8 .', echo=True)
 
 @task
+def benchmark(ctx):
+    """Run benchmarks on codebase."""
+    ctx.run('python performance/benchmark.py', echo=True)
```

Comment:
Minor: If you move the last 3 lines in `benchmark.py` into a `main()` function, you could call it here and avoid shelling out. This would also allow users to import the benchmarking module without automatically running the benchmarks.